src/utils/navigation.ts

```diff
@@ -57,6 +57,11 @@ export const NAVIGATION: Navigation = {
   [config.WHO_IS_MAKING_UPDATE_URL]: {
     currentPage: config.WHO_IS_MAKING_UPDATE_PAGE,
     previousPage: () => config.CONFIRM_OVERSEAS_ENTITY_DETAILS_PAGE,
+    nextPage: [config.UPDATE_DUE_DILIGENCE_PAGE, config.UPDATE_DUE_DILIGENCE_OVERSEAS_ENTITY_PAGE]
+  },
+  [config.UPDATE_DUE_DILIGENCE_OVERSEAS_ENTITY_URL]: {
+    currentPage: config.OVERSEAS_ENTITY_DUE_DILIGENCE_PAGE,
```

Comment:
Is this not UPDATE_OVERSEAS_ENTITY_DUE_DILIGENCE_PAGE?

Reply:
fixed